pom.xml

```diff
@@ -161,6 +161,14 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>1.8</source>
```

Comment:
There is an easier way :) Switch the java.level property to 8 `<java.level>8</java.level>`